src/components/OrgFile/components/Header/index.js

```diff
@@ -305,6 +308,37 @@ ${header.get('rawDescription')}`;
     // window.location.href = mailtoURI;
   }
 
+  handleAddNoteClick() {
+    let input = prompt('Enter a note to add to the header:');
+    if (input !== null) input = input.trim();
+    if (!input) return;
+
+    // Wrap line at 70 characters, see fill-column in "Insert note" window (C-c C-z)
+    const wrappedIndentedInput = this.formatTextWrap(input, 70).replace(/\n(.)/, '\n  $1');
+    // Getting timestamp and template string is impure, hence do it here:
+    const timestamp = moment().format('YYYY-MM-DD HH:MM'); // TODO: use getCurrentTimestamp and export timestamp to text?
```

Comment:
Maybe do it similarly like: https://github.com/200ok-ch/organice/blob/master/src/lib/timestamps.js#L79